patient_portal/get_patient_info.php

```diff
@@ -103,104 +100,121 @@
     DEFINE("COL_POR_USER","portal_username");
     DEFINE("COL_POR_SALT","portal_salt");
     DEFINE("COL_POR_PWD_STAT","portal_pwd_status");
-    $sql= "SELECT ".implode(",",array(COL_ID,COL_PID,COL_POR_PWD,COL_POR_SALT,COL_POR_PWD_STAT))
-          ." FROM ".TBL_PAT_ACC_ON
-          ." WHERE ".COL_POR_USER."=?";
-            $auth = privQuery($sql, array($_POST['uname']));
-            if($auth===false)
-            {
+    $sql = "SELECT " . implode(",", array(
+        COL_ID,
+        COL_PID,
+        COL_POR_PWD,
+        COL_POR_SALT,
+        COL_POR_PWD_STAT
```

Comment:
arrays defined in multiple lines should add a trailing comma.  This is the "long array" format, and makes adding new ones easier (without making a change to existing lines.
-No big deal here, as this is probably a permanently enumerated array.